src/features/click-to-load/assets/ctl-placeholder-block.css

```diff
@@ -0,0 +1,336 @@
+* {
+    font-family: DuckDuckGoPrivacyEssentials, system, -apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto,
+        Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+}
```

Comment:
I think the shadow DOM will still inherit some properties from the host page. This has bitten us in the past, where text would render with the wrong alignment or font style. Not sure if this is already taken care of elsewhere, but it may be worth resetting the shadow properties. Here's a list of the inherited properties: https://stackoverflow.com/questions/5612302/which-css-properties-are-inherited#5612360. I haven't vetted it, just a quick search.